airflow/models/dag.py

```diff
@@ -852,52 +805,17 @@ def update_old_perm(permission: str):
                     updated_access_control[role][permissions.RESOURCE_DAG] = set(perms)
                 else:
                     updated_access_control[role] = perms
-                if permissions.RESOURCE_DAG in updated_access_control[role]:
-                    updated_access_control[role][permissions.RESOURCE_DAG] = {
-                        update_old_perm(perm)
-                        for perm in updated_access_control[role][permissions.RESOURCE_DAG]
-                    }
             elif isinstance(perms, dict):
                 # Not allow new access control format with old FAB versions
                 raise AirflowException(
                     "Please upgrade the FAB provider to a version >= 1.3.0 to allow "
                     "use the Dag Level Access Control new format."
                 )
             else:
-                updated_access_control[role] = {update_old_perm(perm) for perm in perms}
+                updated_access_control[role] = set(perms)
 
         return updated_access_control
 
-    def following_schedule(self, dttm):
```

Comment:
Just leaving a comment here so that it's not accidentally merged.

Reply:
Other PR is merged now, so safe for this one as well

Reply:
Just to be onthe safe side, briefly re-based and want to see it green (again) before merge